presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Commit message:
`Add missing cache invalidate calls to CachingHiveMetastore` is implementation detail, however the more important aspect is user visiable behavior change. Also please do not use `-` in commit message title.

I would write:
```
Invalidate Hive metastore cache on GRANT/REVOKE

- Add missing cache invalidate calls to CachingHiveMetastore
- Fix REVOKE in FileHiveMetastore
```

Reply:
splitted into 2 commits